apps/api-express/src/app/daos/search.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  ParsedArticleParagraphStandalone,
+  ParsedArticle,
+  ParsedArticleParagraph,
+  ArticleParagraphBacksUpClaim,
+} from '@foodmedicine/interfaces';
+import { runScholarsScraper } from '@foodmedicine/scholars-scraper';
+import * as articleParser from '@foodmedicine/article-parser';
+
+export async function findQueryResults(
+  query: string,
+  opts?: {
+    numberOfArticles?: number;
+    maxNumberOfParagraphs?: number;
+  }
+): Promise<ParsedArticleParagraphStandalone[]> {
+  // The query is the impact and the recommendation is left blank
+  const articleHeads = await runScholarsScraper(
+    query,
+    // TODO this is a temporary fix, removing the entire recommendation, impact abstraction should be done
```

Comment:
Do you mean passing in query twice is a temp fix? Could you instead pass in an empty string?

Reply:
Oh yeah, it passed in twice in order to not interfere with the cross feature